README.md

```diff
@@ -17,12 +17,13 @@ either devnet, mainnet, or (recommended) an RPC URL.
 
 ## Deployments
 
-| program           | tag | program ID                                  |
-| ----------------- | --- | ------------------------------------------- |
-| autocrat_v0       | v0.1| metaX99LHn3A7Gr7VAcCfXhpfocvpMpqQ3eyp3PGUUq |
-| autocrat_migrator | v0.1| migkwAXrXFN34voCYQUhFQBXZJjHrWnpEXbSGTqZdB3 |
-| autocrat_v0       | v0  | meta3cxKzFBmWYgCVozmvCQAS3y9b3fGxrG9HkHL7Wi |
-| conditional_vault | v0  | vaU1tVLj8RFk7mNj1BxqgAsMKKaL8UvEUHvU3tdbZPe |
+| program           | tag  | program ID                                  |
+| ----------------- | ---- | ------------------------------------------- |
+| conditional_vault | v0.1 | vAuLTQjV5AZx5f3UgE75wcnkxnQowWxThn1hGjfCVwP |
```

Comment:
this should be v0.2, that's where we're going to pop this

Reply:
Got it